src/generators/android/templates/typewriterUtils.java.hbs

```diff
@@ -0,0 +1,45 @@
+package com.segment.generated;
+
+import com.segment.analytics.Options;
+import java.util.*;
+import java.lang.String;
+import androidx.annotation.Nullable;
+import androidx.annotation.NonNull;
+
+public final class TypewriterUtils {
```

Comment:
Can we make this a `static` class?

Reply:
I assumed we could, but it looks like a top-level class in Java cannot use the modifier `static`. Inner-classes can and should.

Reply:
SGTM!